### DISCUSSION TITLE
Target of URI doesn't exist: 'package:translations/app_localizations.dart'

### DISCUSSION THREAD

**Discussion by omer358** [ferraridamiano/ConverterNOW#193] [Q&A] [answered]
I'm aware that this is not the right place to discuss this problem but I couldn't find a solution to my problem. so I came here because a problem happened with this project when I tried to import it on my computer. Hope that someone could help me figure out why I'm getting this error [my question on stackoverflow](https://stackoverflow.com/questions/74069745/target-of-uri-doesnt-exist-packagetranslations-app-localizations-dart)

**Comment by ferraridamiano** [comment, marked as answer]
Take a look at the readme on how to compile from source code for different platforms. First you have to install Melos with `dart pub global activate melos` and then you have to bootstrap it with `melos bootstrap`. And finally you can compile the project. It's all written in the Readme 😉
